docs/core/logging.md

```diff
@@ -123,6 +130,44 @@ to customise what is logged.
     }
     ```
 
+### Customising  fields in logs
+
+- Utility by default emits `timestamp` field in the logs in format `yyyy-MM-dd'T'HH:mm:ss.SSSZz` and in system default timezone. 
+If you need to customize format and timezone, you can do so by configuring `log4j2.component.properties` and configuring properties as shown in example below:
+
+=== "log4j2.component.properties"
+
+    ```properties hl_lines="1 2"
+    log4j.layout.jsonTemplate.timestampFormatPattern=yyyy-MM-dd'T'HH:mm:ss.SSSZz
+    log4j.layout.jsonTemplate.timeZone=Europe/Oslo
```

Comment:
Is there a way to actually have that default to UTC unless overridden?

Reply:
Rite now the behaviour is consistent with existing implementation, meaning it will pick system default timezone. But with new approach, users have ability to customise timezone if need be, which was not possible before.